airbyte-integrations/connectors/source-zendesk-support/unit_tests/integrations/test_post_comment_votes.py

```diff
@@ -95,13 +95,11 @@ def test_given_403_error_when_read_posts_comments_then_skip_stream(self, http_mo
 
         output = read_stream("post_comment_votes", SyncMode.full_refresh, self._config)
         assert len(output.records) == 0
-
-        info_logs = get_log_messages_by_log_level(output.logs, LogLevel.INFO)
+        assert output.get_stream_statuses("post_comment_votes")[-1] == AirbyteStreamStatus.INCOMPLETE
```

Comment:
Note that this is a change in behavior. It felt weird to silently fail, the reasons were not documented and it felt weird. The 404 case makes a bit more sense but even then... 

Given the low usage for now, I wouldn't add more complexity and just go with the same error handler as the other streams. We can change that very easily later.

Reply:
After further research , the 404 comes from [here](https://github.com/airbytehq/oncall/issues/2379). It is not clear what is the cause unfortunately.

The handling of the 403 comes from [here](https://github.com/airbytehq/airbyte/pull/23035). 

It is weird because both have been migrated to fail by default [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml#L37-L41) and nobody complained so I'll assume we can still fail on these cases too